IlberkEral/hw3/calculator.js

```diff
@@ -0,0 +1,83 @@
+window.onload = function() {

```

Comment:
Faydalandığım kaynakta bu şekilde kullanılmıştı, sonra sayfanın tamamen yüklenmesinden sonra javascript dosyasının çağırılması banada mantıklı geldi. Bu şekilde kullandım ama daha çok sayfa yüklendikten sonra alert veya resimleri sayfa yüklendikten sonra çağırmak için kullanılıyormuş, biraz gereksiz olmuş sanırım. :(